transformers/tests/utils.py

```diff
@@ -1,11 +1,14 @@
 import os
 import unittest
+import tempfile
 
 from distutils.util import strtobool
 
 from transformers.file_utils import _tf_available, _torch_available
 
 
+CACHE_DIR = os.path.join(tempfile.gettempdir(), "transformers_test")
```

Comment:
Is `tempfile.gettempdir()` usually cleared at machine's reboot? (accross ≠ platforms)

But its output is constant across different process executions (otherwise wouldn't actually cache stuff) – correct?

Reply:
Ok for Ubuntu for instance this is helpful: https://askubuntu.com/questions/20783/how-is-the-tmp-directory-cleaned-up/857154#857154

Reply:
Yes, it's supposed to be cleared at reboot.

On some platforms, /tmp isn't the recommended. Try `echo $TMPDIR` on a mac for example.